psi4/driver/procrouting/empirical_dispersion.py

```diff
@@ -187,19 +203,36 @@ def __init__(self, alias, dtype, **kwargs):
             self.citation += "    Smith, D. G. A.; Burns, L. A.; Patkowski, K.; Sherrill, C. D. (2016), J. Phys. Chem. Lett.; 7: 2197"
             self.bibtex = "Grimme:2011:1456"
 
+        elif self.dtype == "-NL":
+            self.description = "    Grimme's -NL (DFT plus  VV10 correlation) "
+            self.citation = "    Hujo, W.; Grimme, S; (2011), J. Chem. Theory Comput.; 7:3866 \n"
+            self.bibtex = "Grimme:2011:3866"
+
         else:
-            raise Exception("Emperical Dispersion type %s not understood." % self.dtype)
+            raise Exception("Empirical Dispersion type %s not understood." % self.dtype)
 
         # 6b) add custom citations if available
         if custom_citation:
             self.citation += "\n    Parametrisation from: \n" + custom_citation
 
+    def excludelist(self):
+        """
+         exclude list used to avoid explicit compute_energy/gradient/hessian calls in proc.py
+        """
+        if self.disp_type=='nl':
+           return True
+        else: 
+           return False
+
     def print_out(self, level=1):
 
         core.print_out("   => %s: Empirical Dispersion <=\n\n" % self.dtype)
         core.print_out(self.description + "\n")
 
         core.print_out(self.citation + "\n\n")
+       
+        if self.disp_type=='nl':
+           return
```

Comment:
Don't we want to print out the B, C params? Or do they get printed out somewhere else anyway?

Reply:
They are printed out in the functional part. Once is enough.